driver/examples/plot_pcolormesh_cube.py

```diff
@@ -112,91 +114,114 @@ def gather_fortran_data_at_klevel(
-            rotated_fortran_data[t, rank, :, :] = temp[rank, :, :].T
-    return rotated_fortran_data
+        ) as f:
+            for t in range(ts_size):
+                fortran_data[t, rank, :, :] = f[var][t, klevel, :, :].T
+    return fortran_data
 
 
-args = parse_args()
-if args.fortran_data_path is not None:
-    if args.fortran_var is None:
-        raise ValueError(
-            "You must specify the variable name to be subtracted in Fortran data."
-        )
-    fortran = gather_fortran_data_at_klevel(
-        args.zarr_output, 6, args.size, args.fortran_var, args.zlevel
-    )
-ds = xr.open_zarr(
-    store=zarr.DirectoryStore(path="/model/output.zarr"), consolidated=False
-)
-python_lat = ds["lat"].values * 180.0 / np.pi
-python_lon = ds["lon"].values * 180.0 / np.pi
-if args.diff_init:
+if __name__ == "__main__":
+    args = parse_args()
     if args.fortran_data_path is not None:
+        if args.fortran_var is None:
+            raise ValueError(
+                "You must specify the variable name (fortran_var) \
+                    to be subtracted in Fortran data."
+            )
+        fortran = gather_fortran_data_at_klevel(
+            args.fortran_data_path, args.size, args.fortran_var, args.zlevel
+        )
+    if args.fortran_var is not None and args.fortran_data_path is None:
         raise ValueError(
-            "You cannot plot the difference from Fortran \
-                when plotting the python difference from the first time step."
+            "You must specify the path (fortran_data_path) to Fortran data."
         )
-    python_init = (
-        ds[args.variable][:, :, 0 : args.size, 0 : args.size, :]
-        .isel(time=0, z=args.zlevel)
-        .values
-    )
-for t in range(args.start, args.stop):
-    python = (
-        ds[args.variable][:, :, 0 : args.size, 0 : args.size, :]
-        .isel(time=t, z=args.zlevel)
-        .values
-    )
-    if args.fortran_data_path is not None:
-        plotted_data = python - fortran
-    elif args.diff_init:
-        plotted_data = python - python_init
-    else:
-        plotted_data = python
-    fig, ax = plt.subplots(1, 1, subplot_kw={"projection": ccrs.Robinson()})
-    if args.vmin is not None and args.vmax is not None:
-        h = pcolormesh_cube(
-            python_lat,
-            python_lon,
-            plotted_data,
-            cmap=plt.cm.bwr,
-            ax=ax,
-            vmin=args.vmin,
-            vmax=args.vmax,
+    if args.zarr_output is not None:
+        ds = xr.open_zarr(
+            store=zarr.DirectoryStore(path=args.zarr_output), consolidated=False
         )
     else:
-        h = pcolormesh_cube(
-            python_lat,
-            python_lon,
-            plotted_data,
-            ax=ax,
+        # when plotting through sarus or docker
+        # we expect pace output mounted at /pace
+        ds = xr.open_zarr(
+            store=zarr.DirectoryStore(path="/pace/output.zarr"), consolidated=False
```

Comment:
we could simplify the logic here and make this the default for the argument. But if you want it here explicitly I'm fine with it